helm-chart/dash0-operator/templates/operator/custom-resource-definition-dash0view.yaml

```diff
@@ -0,0 +1,181 @@
+---
```

Comment:
remark: I assume you used one of the other Helm custom resource types as a blueprint. The way we usually would create and maintain the Helm definitions for custom resources would be Go-code first:
- Describe the resource type in `api/dash0monitoring/v1alpha/dash0view_types.go`
- Run `make generate` to have `config/crd/bases/operator.dash0.com_dash0views.yaml` generated
- Copy that file over to `helm-chart/dash0-operator/templates/operator/custom-resource-definition-dash0view.yaml`

It is a bit cumbersome (due to the fact that the operator framework and tooling works with kustomize instead of Helm), but it ensures that actual instances of the resource type can be properly deserialized at runtime.